src/edie/api.py

```diff
@@ -30,10 +40,12 @@ def list(self, dictionary, limit=None, offset=None):
             url = f"{self.endpoint}list/{dictionary}?{qstr}"
         else:
             url = f"{self.endpoint}list/{dictionary}"
-        return requests.get(url).json()
+        headers = self.__get_header()
+        return requests.get(url, headers=headers).json()
 
     def json(self, dictionary, id):
```

Comment:
I also think we should rename the parameters here to make it more intuitive for the readers. for example, we need here dictionary id and entry id when i understand the api correctly.